src/universal/modules/notifications/ducks/notificationDuck.js

```diff
@@ -0,0 +1,23 @@
+const NOTIFICATION_BAR_PRESENT = 'notifications/NOTIFICATION_BAR_PRESENT';
```

Comment:
where does this bar appear? all dashboards? meeting? i think we may wanna refactor it into a dashboard duck. we already have an entity called notifications & having a notifications duck that doesnt pertain to it is gonna get confusing.